build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -x
+git submodule update --init
+mvn clean install -U -Dmaven.test.skip=true 
```

Comment:
I almost removed this script file, as it's not used for Jenkins which is the correct place to build. So at the moment it's serves as a documented quickstart, to enable another dev picking this up to know what the required git submodule incantations are.